packages/unimodules-app-loader/android/build.gradle

```diff
@@ -0,0 +1,76 @@
+buildscript {
+  repositories {
+    google()
+    jcenter()
+  }
+
+  dependencies {
+    classpath 'com.android.tools.build:gradle:3.5.1'
+  }
+}
```

Comment:
Remove this buildscript block, we no longer declare gradle plugin as a build dependency in our packages